sgp30/sgp30.c

```diff
@@ -251,20 +154,7 @@ int16_t sgp30_measure_test(uint16_t *test_result) {
  * Return:  STATUS_OK on success, an error code otherwise
  */
 int16_t sgp30_measure_iaq() {
-    const struct sgp_profile *profile;
-    int16_t ret;
-
-    profile = sgp30_get_profile_by_number(PROFILE_NUMBER_IAQ_MEASURE);
-    if (profile == NULL)
-        return STATUS_FAIL;
-
-    ret = sensirion_i2c_write_cmd(SGP30_I2C_ADDRESS, profile->command);
-    if (ret != STATUS_OK)
-        return ret;
-
-    client_data.current_state = MEASURING_PROFILE_STATE;
```

Comment:
Did we previously check that the user is using the API correctly? i.e. by verifying the `current_state`? If yes it may be desirable to keep this behavior.

Reply:
We checked if the user called `sgp30_measure_raw` and `sgp30_measure_iaq` before `sgp30_read_raw` and `sgp30_read_iaq`, respectively. However, as mentioned above, this makes it virtually impossible to work with multiple sensors of the same type at the same time.

As a sidenote, there are no such checks in the other embedded drivers. So I'd suggest to either add this check to all embedded drivers or not have the check at all.

Reply:
I added a note to the changelog, that the driver no longer checks if a measurement was started when reading a measurement